Source/Diffusion.cpp

```diff
@@ -73,14 +70,13 @@ PeleC::getMOLSrcTerm(
   const amrex::GpuArray<amrex::Real, AMREX_SPACEDIM> dxD = {
     {AMREX_D_DECL(dx1, dx1, dx1)}};
 
-  // Grab the BCs
+  // BCs
```

Comment:
I think these are only used for EB redistribution so this could be moved down to the EB section

Reply:
another good catch, moved them down to that section